openquake/risklib/riskinput.py

```diff
@@ -290,14 +287,14 @@ def __init__(self, imt_taxonomies, hazard_by_site, assets_by_site,
             self.weight += len(assets)
         self.taxonomies = sorted(taxonomies)
         self.tags = None  # for API compatibility with RiskInputFromRuptures
-        self.eps_dict = eps_dict or {}
+        self.eps = {} if eps is None else eps
```

Comment:
type inconsistency: if eps is None seems to me more correct pass an empty numpy array.